docs/zh/week12/12-3.md

```diff
@@ -42,36 +42,36 @@ $$
 
 请注意，这与我们到目前为止看到的隐藏表示形式不同，在隐藏表示形式中，输入乘以权重矩阵。
 
-根据我们施加在向量$ \ vect {a} $上的约束，我们可以获得硬或软的注意力。
+根据我们施加在向量$\vect{a}$上的约束，我们可以获得硬或软的注意力。
 
 
 #### 硬注意力
 
-格外注意，我们对alpha施加了以下约束：$ \ Vert \ vect {a} \ Vert_0 = 1 $。 这意味着$ \ vect {a} $是一个热门向量。 因此，输入的线性组合中除一个系数外的所有系数都等于零，并且隐藏表示减小为与元素$ \ alpha_i = 1 $相对应的输入$ \ boldsymbol {x} _i $。
+格外注意，我们对alpha施加了以下约束：$\Vert\vect{a}\Vert_0 = 1$。 这意味着$\vect{a}$是一个热门向量。 因此，输入的线性组合中除一个系数外的所有系数都等于零，并且隐藏表示减小为与元素$\alpha_i = 1$相对应的输入$\boldsymbol{x}_i$。
 
 #### 软注意力
 
-对于软注意力，我们强加$ \ Vert \ vect {a} \ Vert_1 = 1 $。 隐藏的表示形式是系数总和为1的输入的线性组合。
+对于软注意力，我们强加$\Vert\vect{a}\Vert_1 = 1$。 隐藏的表示形式是系数总和为1的输入的线性组合。
 
 ### 自我注意力 (II)
 
-$ \ alpha_i $来自哪里？
+$\alpha_i$来自哪里？
 
-我们通过以下方式在\ mathbb {R} ^ t $中获得向量$ \ vect {a} \
+我们通过以下方式获得向量$\vect{a} \in \mathbb{R}^t$
 
 $$
 \vect{a} = \text{[soft](arg)max}_{\beta} (\boldsymbol{X}^{\top}\boldsymbol{x})
 $$
 
-其中$ \ beta $表示$ \ text {soft（arg）max}（\ cdot）$的逆温度参数。 $ \ boldsymbol {X} ^ {\ top} \ in \ mathbb {R} ^ {t \ times n} $是集合$ \ lbrace \ boldsymbol {x} _i \ rbrace \ _ {i = 1} ^ t $，而$ \ boldsymbol {x} $代表集合中的通用$ \ boldsymbol {x} _i $。 请注意，$ X ^ {\ top} $的第$ j $行对应于元素$ \ boldsymbol {x} _j \ in \ mathbb {R} ^ n $，因此$$的$ j $行 \ boldsymbol {X} ^ {\ top} \ boldsymbol {x} $是$ \ boldsymbol {x} _j $与$ \ lbrace中每个$ \ boldsymbol {x} _i $的标量乘积\ bracesymbol {x} _i \ rbrace \ _ {i = 1} ^ t $。
+其中$\beta$表示$\text{soft(arg)max}(\cdot)$的逆温度参数。 $\boldsymbol{X}^{\top}\in\mathbb{R}^{t \times n}$是集合$\lbrace\boldsymbol{x}_i \rbrace\_{i=1}^t$的转置矩阵表示，而$\boldsymbol{x}$代表集合中的通用$\boldsymbol{x}_i$。 请注意，$X^{\top}$的第$j$行对应于元素$\boldsymbol{x}_j\in\mathbb{R}^n$，因此$\boldsymbol{X}^{\top}\boldsymbol{x}$的$j$行是$\boldsymbol{x}_j$与$\lbrace \boldsymbol{x}_i \rbrace_{i=1}^t$中每个$\boldsymbol{x}_i$的标量乘积。
```

Comment:
Here as well, we need `\_`.
Also, you removed `\ bracesymbol {x} _i \ rbrace \ _ {i = 1} ^ t $` entirely.